GUIDE.md

```diff
@@ -203,7 +203,7 @@ ipfs.on('ready', async () => {
 })
 ```
 
-To give write access to another peer, you'll need to get their public key with some means. They'll need to give you the output of their OrbitDB instance's key: `orbitdb.identity.publicKey`.
+To give write access to another peer, you'll need to get their public key with some means. They'll need to give you the output of their OrbitDB instance's key: `orbitdb.identity.id`.
```

Comment:
should probably have changed 'OrbitDB instance's key' to 'OrbitDB instance's id'